docs/src/operate/security/mqtt-tls-configuration.md

```diff
@@ -226,6 +228,33 @@ openssl x509 -req \
     -days 365
 ```
 
+## Username/Password authentication {#username-password-authentication}
+
+When a local MQTT broker requires username/password authentication,
+the %%te%% components also need to know this username and password.
+
+Configure the username and the path to the password file using `tedge config`:
+
+```sh
+sudo tedge config set mqtt.client.auth.username "USERNAME"
+sudo tedge config set mqtt.client.auth.password_file "PATH_TO_PASSWORD_FILE"
+```
+
+In the password file, write the password on the first line. The second line and any subsequent lines will be ignored.
+
+```sh title="file: <file_path>"
+YOUR_PASSWORD
+```
+
```

Comment:
Add one line about restricting the access levels to that file as soon as it is created?

Reply:
> Add one line about restricting the access levels(600) to that file as soon as it is created?

This is a good suggestion, even if this prevent regular users to `tedge mqtt pub/sub` because somehow this is the point to have passwords and private keys.

Reply:
Updated the doc in 27d1ab56e001a7481d1036c6f6d4eccd3c422ef3